### PR TITLE
Add store-tool

### PR DESCRIPTION
**Pull request by sakridge**
#### Problem

No standalone way to inspect account stores offline.

#### Summary of Changes

Add a tool to open account stores and print their contents.

Fixes #

### CODE REVIEW

File: runtime/store-tool/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[package]
+authors = ["Solana Maintainers <maintainers@solana.foundation>"]
+edition = "2018"
+name = "solana-store-tool"
+description = "Tool to inspect append vecs"
+version = "1.5.0"
+repository = "https://github.com/solana-labs/solana"
+license = "Apache-2.0"
+homepage = "https://solana.com/"
+
```
Comment (mvines) [inline runtime/store-tool/Cargo.toml:10]: `publish = false`?

Comment (sakridge) [reply runtime/store-tool/Cargo.toml:11]: added, thanks

File: runtime/store-tool/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,47 @@
+use clap::{crate_description, crate_name, value_t_or_exit, App, Arg};
+use log::*;
+use solana_runtime::append_vec::AppendVec;
+
+fn main() {
```
Comment (ryoqun) [inline runtime/store-tool/src/main.rs:5]: well, there is never-merged similar pr: https://github.com/solana-labs/solana/pull/9490/files#diff-fe20bb9062954767913766205f72e8eb5ae88909c0d2529ec706e27a3e0eb800R589

Comment (sakridge) [reply runtime/store-tool/src/main.rs:5]: and?

**Pull request merged**